docs/source/Support/bskReleaseNotes.rst

```diff
@@ -63,6 +63,8 @@ Version |release|
 - Fixed a bug in eclipse that caused potentially occluding bodies to be skipped if a prior body was closer to the sun than
```

Comment:
Your commit message for this commit is too long. Notice how github truncates it with and ellipsis.

Reply:
Thank you for the tip here. The new commit message should now comply to the length requirements.